libs/fairness/src/lib/Controls/OutcomePlot.tsx

```diff
@@ -61,10 +75,33 @@ export class OutcomePlot extends React.PureComponent<IOutcomePlotProps> {
           y: groupNamesWithBuffer
         }
       ];
+      if (this.props.errorPickerProps.selectedErrorKey !== "disabled") {
+        barPlotlyProps.data[0].error_x = {
+          array: this.props.metrics.outcomes.binBounds?.map(
+            (binBound, index) => {
+              if (this.props.metrics.outcomes?.bins[index] !== undefined) {
+                return binBound.upper - this.props.metrics.outcomes.bins[index]; // convert from bounds to relative error
+              }
+              return 0;
+            }
+          ) || [0],
+          arrayminus: this.props.metrics.outcomes.binBounds?.map(
```

Comment:
The naming here doesn't make it obvious to me what this is. We have array above and arrayminus here. What do these mean? Can you add a comment above to explain? Thanks!

Reply:
array and arrayminus are from the Plotly API

Reply:
Sure, but it's still not obvious. 🙂 Believe me, anyone reading this in 3 months will be happy for any hint.